### PR TITLE
Dyn 4936 watch node automatic height

### PR DESCRIPTION
**Pull request by jesusalvino**
### Purpose

Set the proper height to a WatchNode when it's dealing with a multiline string source

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers
@QilongTang

### FYIs
@RobertGlobant20 @filipeotero

### CODE REVIEW

**Comment by jesusalvino** [comment]
imagen

imagen

File: src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs [hunk 1]
```diff
@@ -56,6 +57,13 @@ private void _vm_PropertyChanged(object sender, System.ComponentModel.PropertyCh
                 else
                 {
                     this.Height = minHeightSize;
+                    if (_vm.Children.Count !=0)
+                    {
+                        if (_vm.Children[0].NodeLabel.Contains(Environment.NewLine))
+                        {
+                            this.Height = defaultHeightSize;
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs:64]: In which case is `defaultHeightSize` still used? Should we count the number of `Environment.NewLine` and then multiply `minHeightSize` to be more accurate?

Comment (jesusalvino) [reply src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs:64]: defaultHeightSize is used when the node display a collection / the second question depends on the Business, I guess Aabishkar KC could help us.

Comment (avidit) [reply src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs:64]: @Amoursol  or @Jingyi-Wen  would know better

Comment (QilongTang) [reply src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs:64]: @jesusalvino Then is minHeight including the watch node header? If yes, maybe this is good enough for now

**Review by QilongTang — APPROVED** [review]

**Pull request merged**